### ISSUE TITLE
OB-6 patches come with surprising names when they were started from a factory patch

### ISSUE THREAD

**Issue by christofmuc** [christofmuc/KnobKraft-orm#20]
This is not really a bug, but a workflow issue - if you create a new patch based on any other patch in the OB-6, and import it into the computer, it will still have the original name. Because the OB-6 has no display for it, and no way to rename it, that can be quite suprising. 

What should I do? If there is already a patch with that name, offer to rename? Offer to call it "BasicProgram"? That's the name you get when you start with an init patch on the OB-6.

**Comment by christofmuc** [comment]
I could build a rename function - if a new patch comes in, but the name is already used, I could generate a new unique name (e.g. appending a number). That would at least show the original patch as it does not contain any postfix?

**Comment by christofmuc** [comment]
I could allow the user to define his or her short sign to automatically append, marking patches as their own?

**Comment by markusschloesser** [comment]
I suggest v1, v2, cos short, or for synths that are numerical only .1 .2  
Normally I'd like date as well, but that should probably be in meta data

**Comment by christofmuc** [comment]
Or a settable short sign, like "Hot Factory Patch CR.1" to indicate I changed it and this is the first iteration. Then when you modify and redo get edit buffer you'll get .2 .3 .4...

**Comment by markusschloesser** [comment]
Settable is a great idea! 😊
But then kk needs to know what a synths is capable of w re to patch names

**Comment by christofmuc** [comment]
> 
> 
> Settable is a great idea! 😊
> But then kk needs to know what a synths is capable of w re to patch names

True, or we need to distinguish between database name and patch-internal name. Keeping those in sync currently also does not really work, and ideally patches are identified via fingerprint and not via name, so we could do automatic synth-patch renaming like back in the Windows 8.3 filename days, like automatic shorten. That truly depends on which UI you use most. For the OB-6, it is really hilarious in that you can embed a 20 character name of a patch in the data, but the OB has only the LED 7-segment display :)
